linera-base/src/data_types.rs

```diff
@@ -1156,6 +1156,34 @@ impl<'a> Deserialize<'a> for Blob {
     }
 }
 
+/// An event recorded in an executed block.
+#[derive(Debug, PartialEq, Eq, Hash, Clone, Serialize, Deserialize, SimpleObject)]
+pub struct EventRecord {
```

Comment:
Why not just `Event`?

Reply:
Right… the original reason (from the original PR) was that we already had an `Event` type with a different meaning. But now I can rename it…